backend/de.metas.contracts/src/main/java/de/metas/contracts/modular/log/ModularContractLogDAO.java

```diff
@@ -136,6 +150,21 @@ public ModularContractLogEntryId create(@NonNull final LogEntryCreateRequest req
 	@NonNull
 	private ModularContractLogEntry fromRecord(@NonNull final I_ModCntr_Log record)
 	{
+		final ModuleConfigId moduleConfigId = Optional.of(record.getModCntr_Module_ID())
+				.filter(moduleId -> moduleId > 0)
+				.map(moduleId -> InterfaceWrapperHelper.load(record.getModCntr_Module_ID(), I_ModCntr_Module.class))
```

Comment:
IMHO you are breaking the boundaries of this repository.
I_ModCntr_Module shall be loaded by ModularContractSettingsDAO.

Question: why do you need configId?

Reply:
We don't need it. I just wanted to keep a kind of consistency between the model (`ModularContractLogEntry`) and request (`LogEntryCreateRequest`). Removed it (along with `ModularContractTypeId`) from the model.